charts/aruba-uxi/values.example.yaml

```
@@ -74,6 +74,10 @@ aruba-uxi:
         name: service-account-name
         ## Any annotations to add the service account that is created (optional, default: {})
         annotations: {}
+      ## Adds the specified annotations to the metadata of the pods
+      podAnnotations:
+        example/annotation: 'foo'
+        example/annotation.for.spirit: 'bar'
```

```suggestion
      podAnnotations: {}
```
Do you think setting it to the default value looks better or conveys the way its used?

---

I like this better